resources/main.css

```diff
@@ -881,5 +873,88 @@ dl dd {
 }
 
 #mw-input-wpscratchwikiskin-header-color {
-	font-family: monospace;
+    font-family: monospace;
+}
+
+/* Mobile design */
+
+@media only screen and (max-width: 405px) {
+    #navigation .inner .tips,
+    #footer .inner .legal {
+        display: none;
+    }
+}
+
+@media only screen and (max-width: 519px) {
+    .inner {
+        width: 100%;
+    }
+    #navigation .inner .discuss,
+    #navigation .inner .explore,
+    #navigation .inner .search,
```

Comment:
since searching is a vital part of the wiki, it should always be available. It could appear somewhere else than in the menu. Maybe have just the search symbol and the search fills the screen when clicked on it?

Reply:
Good point, I did it this way because Scratch does it this way. I'll hide something else.